meveo-model/src/main/java/org/meveo/model/security/Permission.java

```diff
@@ -49,13 +52,14 @@
  */
 @Entity
 @Cacheable
+@ModuleItem(path = "Permission", value = "Permissions")
```

Comment:
Value should be "Permission" and path "permission"

Reply:
Rectified